test/unit/org/apache/cassandra/nodes/virtual/LegacySystemKeyspaceToNodesTest.java

```diff
@@ -91,6 +91,7 @@ public class LegacySystemKeyspaceToNodesTest extends CQLTester
     @BeforeClass
     public static void setup() throws Exception
     {
+        DatabaseDescriptor.setAutoSnapshot(true);
```

Comment:
why is that change needed now?

Reply:
Because I've disabled auto snapshot for tests by default